semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/operatorextension/Nodes.scala

```diff
@@ -3,7 +3,7 @@ package io.shiftleft.semanticcpg.language.operatorextension
 import io.shiftleft.codepropertygraph.generated.nodes.Call
 
 object opnodes {
-  class Assignment(call: Call) extends Call(call.graph, call.id)
-  class Arithmetic(call: Call) extends Call(call.graph, call.id)
-  class ArrayAccess(call: Call) extends Call(call.graph, call.id)
+  case class Assignment(val call: Call)
```

Comment:
```suggestion
  case class Assignment(call: Call)
```
case class members are publicly accessible by default, same for the lines below